content/docs/esc/environments/webhooks.md

```diff
@@ -39,13 +39,43 @@ Organization webhooks can be managed on the Organization Settings page. Environm
+Pulumi Cloud also supports webhooks for events related to Pulumi IaC stacks and [Pulumi Deployments](/docs/pulumi-cloud/deployments). For additional information on these types of webhooks, see [Pulumi Cloud Webhooks](/docs/pulumi-cloud/webhooks).
 
 ### Create a Webhook
 
-Pulumi Webhooks may be created through the UI using the steps outlined below, by using the
-[Webhook resource](https://www.pulumi.com/registry/packages/pulumiservice/api-docs/webhook/) from the Pulumi provider
-or by [using the API](/docs/pulumi-cloud/cloud-rest-api/#create-webhook) directly.
+Pulumi Webhooks may be created through any of the following methods:
+
+1. Manually, in the Pulumi Cloud UI using the steps outlined in [Create an Organization Webhook in the Pulumi Cloud UI](#create-an-organization-webhook-in-the-pulumi-cloud-ui) or [Create an Environment Webhook in Pulumi Cloud in the Pulumi Cloud UI](#create-an-environment-webhook-in-the-pulumi-cloud-ui).
+1. Declaratively, as part of a [Pulumi IaC](/docs/iac) program as shown in [Create an Webhook as Part of a Pulumi IaC Program](#create-an-webhook-as-part-of-a-pulumi-iac-program)
+1. By invoking the [Pulumi Cloud REST API](/docs/pulumi-cloud/cloud-rest-api/#create-webhook) directly.
+
+#### Create an Organization Webhook in the Pulumi Cloud UI
+
+1. Navigate to **Settings** > **Webhooks**.
+2. Select **Create webhook**.
+3. Under Destination, choose **Webhook**, **Slack** or **Microsoft Teams**.
+    1. For generic JSON webhooks, provide a display name, payload URL, and optionally a secret.
+    2. For Slack webhooks, provide a Slack webhook URL and a display name.
+    3. For Microsoft Teams webhooks, provide a Microsoft Teams webhook URL and a display name.
+4. Choose which events you would like to receive using groups and filters menu.
+
+#### Create an Environment Webhook in the Pulumi Cloud UI
+
+1. Navigate to your environment.
+2. Navigate to **Webhooks** tab.
+3. Select **Create webhook**.
+4. Under Destination, choose **Webhook**, **Slack**, **Microsoft Teams** or **Deployment**
+   1. For generic JSON webhooks, provide a display name, payload URL, and optionally a secret.
+   2. For Slack webhooks, provide a Slack webhook URL and a display name.
+   3. For Microsoft Teams webhooks, provide a Microsoft Teams webhook URL and a display name.
+   4. For Deployment webhooks, provide the stack to deploy in the format `project/stack`.
+5. Choose which events you would like to receive using groups and filters menu.
+
+#### Create an Webhook as Part of a Pulumi IaC Program
+
+The following example shows how to create an Environment webhook in a Pulumi IaC program by declaring a [Webhook resource](/registry/packages/pulumiservice/api-docs/webhook/) with the [Pulumi Cloud provider](/registry/packages/pulumiservice).
+
+To create an Organization webhook instead of an environment webhook, the code is virtually identical - just omit the `environmentName` value when declaring the webhook resource.
```

Comment:
```suggestion
To create an Organization webhook instead of an Environment webhook, the code is virtually identical - just omit the `environmentName` value when declaring the webhook resource.
```